apps/mobile/src/components/ProfileView/ProfileViewHeader.tsx

```diff
@@ -88,13 +88,12 @@ export function ProfileViewHeader({ queryRef, selectedRoute, onRouteChange }: Pr
   return (
     <View>
       {user.bio && (
-        <View className="mb-4 px-4">
+        <View className="px-4">
           <Markdown>{user.bio}</Markdown>
         </View>
       )}
       {!isLoggedInUser && <ProfileViewSharedInfo userRef={user} />}
-
-      <View className="flex flex-row mx-4 space-x-12 ">
+      <View className={`flex flex-row mx-4 space-x-12 ${isLoggedInUser ? 'mt-2.5' : 'mt-4'}`}>
```

Comment:
instead of using `isLoggedInUser` to adjust the margin on this element, I think the absence of `ProfileViewSharedInfo` if `!isLoggedInUser` on line 95 should dictate the amount of extra space

so in other words the extra space should be on the conditional ProfileViewSharedInfo , or be set as a gap on the parent so that there's automatically more space if `ProfileViewSharedInfo` is present, and less space if `ProfileViewSharedInfo` is not present